extension/controller/experiments/TranslationBar/api.js

```diff
@@ -5,7 +5,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-/* global ExtensionAPI, ChromeUtils, modelRegistry, TranslationNotificationManager  */
+/* global ExtensionCommon, ExtensionAPI, ChromeUtils, Services, modelRegistry, TranslationNotificationManager  */
+
+// the Services object was not available until Firefox 88 - bugzil.la/1698158
```

Comment:
According to manifest.json, the supported min version is Firefox 79, so I have included this for compatibility with Firefox 87 and earlier. If such (1+ year) old Firefox versions are not supported, then this can be removed.